subsys/partition_manager/pm.yml.trustzone

```diff
@@ -4,5 +4,9 @@ sram_secure:
   region: sram_primary
 
 sram_nonsecure:
-  span: [sram_primary] /* Here, sram_primary refers to the (NS) app's RAM. */
   region: sram_primary
+  span:
+    - sram_primary /* Here, sram_primary refers to the (NS) app's RAM. */
+#ifdef CONFIG_BT_RPMSG_NRF53
```

Comment:
the condition seems to be not sufficient, look at the related changes: https://github.com/nrfconnect/sdk-nrf/pull/6672/files

Reply:
I've now removed this ifdef.